client/post-editor/media-modal/image-editor/image-editor-canvas.jsx

```diff
@@ -97,12 +100,22 @@ const MediaModalImageEditorCanvas = React.createClass( {
 
 		this.drawImage();
 		this.updateCanvasPosition();
+		this.onWindowResize = throttle( this.updateCanvasPosition, 200 );
+		if ( window ) {
```

Comment:
Oh, great catch. Sorry it slipped me.. `window` is just a variable like any other and we can't test whether it's defined or not like we check for falsey value. Good to know.